src/server/services/cms/cms-content.ts

```diff
@@ -260,25 +261,48 @@ async function getFooter(
     transformResponse: transformFooterResponse,
   });
 
-  return requestData<CMSFooterContent>(requestConfig, requestID).then(
-    (apiData) => {
-      if (apiData.content?.blocks.length) {
-        fileCache.setKey('CMS_CONTENT_FOOTER', apiData);
-        fileCache.save();
-        return apiData;
-      }
-      // Try to get stale cache instead.
-      const staleApiData =
-        fileCache.getKeyStale<ApiResponse_DEPRECATED<CMSFooterContent>>(
-          'CMS_CONTENT_FOOTER'
-        );
-
-      if (staleApiData) {
-        return staleApiData;
-      }
+  async function saveCacheAndSend(
+    apiData: any
+  ): Promise<ApiResponse_DEPRECATED<CMSFooterContent | null>> {
+    if (apiData.content?.blocks.length) {
+      fileCache.setKey('CMS_CONTENT_FOOTER', apiData);
+      fileCache.save();
+      return apiData;
+    }
+    // Try to get stale cache instead.
+    const staleApiData =
+      fileCache.getKeyStale<ApiResponse_DEPRECATED<CMSFooterContent>>(
+        'CMS_CONTENT_FOOTER'
+      );
 
-      return apiErrorResult('Could not fet Footer data', null);
+    if (staleApiData) {
+      return staleApiData;
     }
+
+    return apiErrorResult('Could not fetch Footer data', null);
+  }
+
+  if (!FeatureToggle.cmsFooterActive) {
```

Comment:
Misschien een nit maar is het niet duidelijker om deze iets van
```suggestion
if (FeatureToggle.useCMSFooterBackupActive) {
```
te noemen?